### PR TITLE
navbar commit test

### PR DESCRIPTION
**Pull request by HornyCapacitor**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
```
Comment (rdycld) [inline src/App.css:2]: esssa

File: src/App.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import { useState } from 'react'
 import './App.css'
+import {Navbar} from "./components/Navbar.jsx"
 
 function App() {
   return (
     <>
-      
+      <Navbar />
```
Comment (rdycld) [inline src/App.jsx:7]: nice changes

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function Navbar() {
+  return (
+    <div className="navbar">
+
```
Comment (rdycld) [inline src/components/Navbar.jsx:4]: fix that

**Review by rdycld — CHANGES_REQUESTED** [review]

File: src/components/Navbar.jsx (hunk 3 above)
Comment (HornyCapacitor) [reply src/components/Navbar.jsx:4]: i will do

**Review by rdycld — APPROVED** [review]

**Pull request merged**